src/field_theory/is_alg_closed/classification.lean

```diff
@@ -176,10 +176,12 @@ begin
   exact ⟨equiv_of_transcendence_basis _ _ e hs ht⟩
 end
 
-private lemma ring_equiv_of_cardinal_eq_of_char_p (p : ℕ) [fact p.prime]
-  [algebra (zmod p) K] [algebra (zmod p) L] (hK : ℵ₀ < #K) (hKL : #K = #L) : K ≃+* L :=
+private def ring_equiv_of_cardinal_eq_of_char_p (p : ℕ) [fact p.prime]
+  [char_p K p] [char_p L p] (hK : ℵ₀ < #K) (hKL : #K = #L) : K ≃+* L :=
 begin
   apply classical.choice,
```

Comment:
This looks better to me
```suggestion
private lemma ring_equiv_of_cardinal_eq_of_char_p (p : ℕ) [fact p.prime]
  [char_p K p] [char_p L p] (hK : ℵ₀ < #K) (hKL : #K = #L) : nonempty (K ≃+* L) :=
begin
```
It's private anyway, and the one place that uses it actually just needs `nonempty _`

Reply:
You mean to keep it a lemma?

Reply:
Yes, but change the statement to include a `nonempty` instead.

Reply:
Done. I've modified also the other statement to use `nonempty` (they had the same problem, with a `@[nolint def_lemma]`).